tests/integration/devfile/cmd_devfile_push_test.go

```diff
@@ -1,5 +1,6 @@
 package devfile
```

Comment:
I guess this file is no longer necessary. Shouldn't we remove it then?
Also, how about also removing the corresponding `test-cmd-devfile-push` Makefile target?

Reply:
I would like to rework some of these tests with `odo dev`

Reply:
Ok, I see. Then maybe a simple comment in the file could help keep track of this?

Reply:
I'll make the changes in this PR, when #5569 is merged